src/controller/ReadInteraction.h

```diff
@@ -87,11 +87,13 @@ template <typename DecodableAttributeType>
 CHIP_ERROR ReadAttribute(Messaging::ExchangeManager * exchangeMgr, const SessionHandle & sessionHandle, EndpointId endpointId,
                          ClusterId clusterId, AttributeId attributeId,
                          typename TypedReadAttributeCallback<DecodableAttributeType>::OnSuccessCallbackType onSuccessCb,
-                         typename TypedReadAttributeCallback<DecodableAttributeType>::OnErrorCallbackType onErrorCb)
+                         typename TypedReadAttributeCallback<DecodableAttributeType>::OnErrorCallbackType onErrorCb,
+                         bool fabricFiltered = false)
```

Comment:
I do wonder whether the default behavior here should be `true`....  What was the reason for picking `false`?

Reply:
It is because previous read requests are all send with fabricFiltered flag = false.
No special reasons besides this.

Reply:
I think we should do `true` as the default behavior, since that will be more common.  People who want to opt into the less-common thing would pass `false` explicitly.

Reply:
Checked the spec that IsFabricFiltered is not an optional field. Updated.